src/style/_ProfileAvatar.scss

```diff
@@ -0,0 +1,62 @@
+#body {
+  .profile-avatar {
+    .containers {
+      margin-bottom: 16px;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      .contact-icon {
+        width: 24px;
+        margin-right: 8px;
+      }
+    }
+
+    .non-verified-profile-pic {
+      border: solid 1px;
+      width: 133px;
+      height: 133px;
+      border-radius: 100%;
+      border-color: black;
+
+      .non-verified-img {
+        width: 100%;
+        height: 100%;
+        border-radius: 100%;
+      }
+    }
+
+    .verified-img-container {
+      .verified-border {
+        width: 136.5px;
```

Comment:
how can .5px be rendered?

Reply:
Good question (copy paste).  I'm going to remove this since it's not part of the design anymore